source/mainnet/smart-contracts/tutorials/sponsoredTransactions/walletConnect.rst

```diff
@@ -0,0 +1,8 @@
+.. include:: ../../../variables.rst
+.. _walletConnect:
+
+=============
+WalletConnect
+=============
+
+Coming soon.
```

Comment:
Could we instead state here that walletconnect does not support the required signMessage yet. And this part will be updated once the wallets do support it.